source/deployment/sso-ldap.rst

```diff
@@ -90,167 +98,187 @@ Using filters assigns roles to specified users on login. To access AD/LDAP filte
 
 When the user accesses the Mattermost URL, they log in with same username and password that they use for organizational logins.
 
-Filters can also be used for excluding users who belong to certain groups. For Active Directory, the query to filter out groups is `(&(memberof=cn=ACME_ALL,ou=Users,dc=sademo,dc=com)(!(memberof=cn=DEV_OPS,ou=Users,dc=sademo,dc=com)))`.
+Filters can also be used for excluding users who belong to certain groups. For Active Directory, the query to filter out groups is ``(&(memberof=cn=ACME_ALL,ou=Users,dc=sademo,dc=com)(!(memberof=cn=DEV_OPS,ou=Users,dc=sademo,dc=com)))``.
 
-##### Guest Filter
+Guest Filter
+''''''''''''
 
 (Optional) When enabled, the Guest Filter in Mattermost identifies external users whose AD/LDAP role is guest and who are invited to join your Mattermost server. These users will have the Guest role applied immediately upon first sign-in instead of the default member user role. This eliminates having to manually assign the role in the System Console.
 
 If this filter is removed/changed, active guests will not be promoted to a member and will retain their Guest role. Guests can be promoted in **System Console > User Management**.
 
-1. Navigate to **System Console > Authentication > Guest Access (Beta)** and set Guest Access to `true`.
+1. Navigate to **System Console > Authentication > Guest Access (Beta)** and set Guest Access to ``true``.
 2. Navigate to **System Console > Authentication > AD/LDAP**.
 3. Complete the **Guest Filter** field.
 4. Choose **Save**.
 
 When a guest logs in for the first time they are presented with a default landing page until they are added to channels.
 
-See the [Guest Accounts documentation](https://docs.mattermost.com/deployment/guest-accounts.html) for more information about this feature.
+See the `Guest Accounts documentation <https://docs.mattermost.com/deployment/guest-accounts.html>`_ for more information about this feature.
 
-##### Admin Filter
+Admin Filter
+''''''''''''
```

Comment:
When clicking View File, this page/file looks broken, maybe something with the header formatting here and in other places needs fixing.

Reply:
Thanks, @amyblais, that's a good point. I actually meant to remove this change from the PR as it's no longer needed. I'll sort that out. :)